### PR TITLE
Handle expired short url

### LINKED ISSUE #8: handle expired short url

### PR DESCRIPTION
**Pull request by yosefbennywidyo**
fixes #8

### CODE REVIEW

File: app/controllers/dynamic_links/redirects_controller.rb [hunk 1]
```diff
@@ -2,13 +2,11 @@ module DynamicLinks
   class RedirectsController < ApplicationController
     def show
       short_url = params[:short_url]
-      link = ShortenedUrl.find_by(short_url: short_url)
+      link = ShortenedUrl.find_by!(short_url: short_url)
```
Comment (saiqulhaq) [inline app/controllers/dynamic_links/redirects_controller.rb:5]: kenapa pake `find_by!`?

Comment (yosefbennywidyo) [reply app/controllers/dynamic_links/redirects_controller.rb:5]: @saiqulhaq supaya langsung raise [ActiveRecord::RecordNotFound](https://api.rubyonrails.org/v7.1.3.2/classes/ActiveRecord/RecordNotFound.html) kalau tidak ada record yang ditemukan.

Comment (yosefbennywidyo) [reply app/controllers/dynamic_links/redirects_controller.rb:5]: > kenapa pake `find_by!`?

Supaya langsung raise `ActiveRecord::RecordNotFound` ketika tidak ada record yang ditemukan

**Review by saiqulhaq — COMMENTED** [review]
for the current implementation, there is no any code who updates `expires_at` column
so we need to implement it at https://github.com/saiqulhaq/dynamic_links/blob/e40231f69b21c7aa3bc0bed157678860d0d8f88d/lib/dynamic_links/shortener.rb#L19 

the default expiry date should be configurable

**Review by saiqulhaq — APPROVED** [review]

**Pull request merged**